docs/application/tizen-studio/rt-ide/overview.md

```diff
@@ -1,24 +1,15 @@
-# Tizen RT
+# TizenRT
 
-Tizen has commercialized in smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The objective of Tizen RT is to extend the Tizen platform device coverage to these kind of low-end devices.
+Tizen has commercialized in smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The objective of TizenRT is to extend the Tizen platform device coverage to these kind of low-end devices.
```

Comment:
Suggest modifying first line to:

Tizen platform is commercialized on smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances.

Reply:
A minor change (make TV and smartphone plural)...
Tizen platform is commercialized on smart TVs, smartphones, wearable devices (Gear S, Gear Fit), and smart home appliances.

Reply:
Ok. i will fix. Thanks.